mhcflurry/class1_affinity_prediction/class1_affinity_predictor.py

```diff
@@ -119,6 +119,27 @@ def supported_peptide_lengths(self):
             max(lower for (lower, upper) in length_ranges),
             min(upper for (lower, upper) in length_ranges))
 
+    def merge(self, affinity_predictors=[]):
+        """ 
+        Merge multiple affinity predictors into one.  
+        When training MHCFlurry in parallel, the user might be left with
```

Comment:
Not really needed to get into the parallel use case here I think. Or could say something like "Used for merging predictors trained in parallel into a single ensemble."